test/prob/gamma/gamma_ccdf_log_test.hpp

```diff
@@ -32,11 +32,11 @@ class AgradCcdfLogGamma : public AgradCcdfLogTest {
     ccdf_log.push_back(
         std::log(1.0 - 0.6321205588285576659757));  // expected ccdf_log
 
-    param[0] = 16.0;  // y
-    param[1] = 3.0;   // alpha
-    param[2] = 3.0;   // beta
+    param[0] = 9.0;  // y
+    param[1] = 1.2;  // alpha
+    param[2] = 1.2;  // beta
     parameters.push_back(param);
-    ccdf_log.push_back(std::log(1.7116220633718619e-18));  // expected ccdf_log
+    ccdf_log.push_back(-10.221534317077268);  // expected ccdf_log
```

Comment:
I did not check this number. Where did you get it from?

Reply:
It's the gamma log ccdf. log ccdf is `log(1.0 - cdf)`.

So in r we can do:
```
log(1.0 - pgamma(9.0, shape = 1.2, rate = 1.2))
```

`beta` in Stan is inverse scale which is the `rate` parameter in R's `pgamma` (https://stat.ethz.ch/R-manual/R-devel/library/stats/html/GammaDist.html). `pgamma` is the CDF of the Gamma pdf in R.